docs/docs/faq.md

```diff
@@ -30,17 +30,10 @@ There are drop-in replacements for `distributed`, `dask`, and `dask-gateway` wit
 
 ## What packages are needed in your environment to create a dashboard?
 
-:::info
-CDS Dashboards has been deprecated in version `2023.9.1`.
-:::
-
-When deploying an app via CDS Dashboards, you will always need to have `cdsdashboards-singleuser` installed in your environment. This allows your environment to show up on the environment options menu when creating your app.
-
-Furthermore, with each framework, you will need to make sure that the specific framework you are deploying is installed. You will need to install `streamlit` for a streamlit app, `panel` for a panel app, etc.
-
-Finally, when creating your app, make note of the packages you are using to run the app locally. These will also need to be added to your environment you will be using to create the dashboard.
-
-There are several example dashboards to build in the dashboard_examples folder on Nebari. The `environment.yml` file in this folder can be used to build an environment that can run each of the dashboard examples available.
+When deploying an app with JHub App Launcher, you need to have the following in your environemtn:
+- `jhub-apps` package
```

Comment:
Ideally we would have jhub-apps-client for this, which when we have it released, should package the minimal things like `jhsingle-native-proxy`, `bokeh-root-cmd` and a few, but it's fine for now.